src/store/holochain.js

```diff
@@ -87,14 +87,14 @@ const initializeClientHolo = async (commit, dispatch, state) => {
 // commit, dispatch and state (unused) here are relative to the holochain store, not the global store
 const initializeClientLocal = async (commit, dispatch, _) => {
   try {
-    const holochainClient = await AppWebsocket.connect(WEB_CLIENT_URI, signal =>
+    const holochainClient = await AppWebsocket.connect(WEB_CLIENT_URI, RECONNECT_SECONDS * 1000, signal =>
```

Comment:
Using `RECONNECT_SECONDS * 1000` here seems misleading since this argument represents a timeout not a reconnect interval